process/sync/shardblock.go

```diff
@@ -487,6 +488,17 @@ func (boot *ShardBootstrap) doJobOnSyncBlockFail(hdr *block.Header, err error) {
 			log.Info(errNotCritical.Error())
 		}
 	}
+
+	if err == process.ErrInvalidBlockHash {
```

Comment:
explanation needed here/

Reply:
Explained above, and I have put a short comment in a TODO there

Reply:
please add to bugs and changes slack channel as well.